cdap-ui/app/hydrator/controllers/create/toppanel-ctrl.js

```diff
@@ -341,7 +371,7 @@ class HydratorPlusPlusTopPanelCtrl {
     this.previewRunning = true;
     let poll = this.dataSrc.poll({
       _cdapNsPath: '/previews/' + previewId + '/status',
-      interval: 5000
+      interval: 500
```

Comment:
500 might be too fast... since the interval for the counter is 1 second, I think you can just make this `1000`. Unless I'm missing something for making this `500`?